core/src/test/java/io/questdb/cairo/OrderByAscRowSkippingTest.java

```diff
@@ -615,6 +622,86 @@ public void test_partitionPerRow_select_last_N_ordered_by_nonTs_column() throws
         assertQuery("l\n8\n9\n10\n", "select l from tab order by l asc limit -3");
     }
 
+    //tests "partitionCount < 1" conditional in FullFwdDataFrameCursor.skipTo()
+    @Test
+    public void test_skip_over_empty_table_with_no_partitions_returns_no_rows() throws Exception {
```

Comment:
could we please maintain CamelCase test naming for consistency ?

Reply:
done